package.json

```
@@ -33,7 +33,8 @@
   "scripts": {
     "build:cli": "rimraf ./bin && tsc --project ./cli/tsconfig.json && chmod u+x ./bin/index.js",
     "build:docs": "vitepress build docs",
-    "build:kongponents": "pnpm typecheck && vite build && vue-tsc -p './tsconfig.build.json' --emitDeclarationOnly && tsc-alias -p './tsconfig.build.json' && pnpm build:nuxt",
+    "build:kongponents:umd": "pnpm typecheck && BUILD_UMD=true vite build && vue-tsc -p './tsconfig.build.json' --emitDeclarationOnly && tsc-alias -p './tsconfig.build.json'",
```

We shouldn't need to build all the types again, etc. Will something like this work instead?
```suggestion
    "build:kongponents:umd": "BUILD_UMD=true vite build",
```